test/util/IdTableHelpers.h

```diff
@@ -87,11 +85,10 @@ void compareIdTableWithExpectedContent(
   auto writeIdTableToStream = [&traceMessage](const IdTable& idTable) {
     std::ranges::for_each(idTable,
                           [&traceMessage](const auto& row) {
-                            // TODO Could be done in one line, if row was
-                            // iterable. Unfortunaly, it isn't.
-                            for (size_t i = 0; i < row.numColumns(); i++) {
-                              traceMessage << row[i] << " ";
-                            }
+                            std::ranges::for_each(row,
+                                [&traceMessage](const auto& rowEntry){
+                                traceMessage << rowEntry << " ";
+                                });
```

Comment:
Add a `TODO<C++23> Use `std::views::join_with` for both loops`.

Reply:
Done, see [656b045](https://github.com/ad-freiburg/qlever/pull/865/commits/656b04510783a0dc95d39adf4dce34af4cc3646c).